cosmos/operators/aws_eks.py

```diff
@@ -160,3 +161,12 @@ class DbtRunOperationAwsEksOperator(DbtAwsEksBaseOperator, DbtRunOperationKubern
 
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
+
+
+class DbtCloneAwsEksOperator(DbtAwsEksBaseOperator, DbtCloneKubernetesOperator):
+    """
+    Executes a dbt core clone command.
+    """
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
```

Comment:
What value is the constructor giving us? Would it make sense to do something like this:

```suggestion
class DbtCloneAwsEksOperator(DbtAwsEksBaseOperator, DbtCloneKubernetesOperator):
    """
    Executes a dbt core clone command.
    """
    pass
```

Reply:
@tatiana We introduced init in PR [#1162](https://github.com/astronomer/astronomer-cosmos/pull/1162) because the operator import issue was observed for Airflow version 2.10 + Cosmos, and [#1161](https://github.com/astronomer/astronomer-cosmos/issues/1161) was reported in Cosmos. Do you think we should keep this in this PR, and when we work on [#1161](https://github.com/astronomer/astronomer-cosmos/issues/1161), we can clean it up then? That way, we'll be more comfortable.

Reply:
There was an alternative solution proposed in PR https://github.com/astronomer/astronomer-cosmos/pull/1163, but it started breaking other tests. This may require some investigations.

Reply:
Let's not block the current ticket due to this. I'm happy for us to review this approach in #1161 , as you suggested, @pankajastro !